src/js/styles/generateStyles.js

```diff
@@ -164,20 +166,20 @@ export function generateStyles(options) {
     '&[x-placement^="top"]': {
       marginBottom: arrowMargin,
 
-      '.tippy-arrow': {
+      [`.${tippyPrefix}tippy-arrow`]: {
```

Comment:
While maintaining a separate `classPrefix` and `tippyPrefix` is technically okay, I'm curious, do you have a need for them to be different or could they both be `demo-`? etc

Reply:
I guess practically they would be the same, but we still need option to choose if to prefix tippy or not (separate from the shepherd prefix)